### PR TITLE
Cleanup after cortina merge, reduce diff with avax

### PR DESCRIPTION
**Pull request by evlekht**
This and https://github.com/chain4travel/caminogo/pull/371 are cleanup PRs after cortina merges.
It removes empty-line diff with avax, unnecessary c4t headers, adds missing c4t headers and also adds missing camino prefixes to filenames.

## Additional references
Original PR based on cortina-15 dev
https://github.com/chain4travel/caminoethvm/pull/103

### CODE REVIEW

File: internal/ethapi/backend.go [hunk 1]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2024, Chain4Travel AG. All rights reserved.
```
Comment (mo-c4t) [inline internal/ethapi/backend.go:1]: year is not correct for c4t license header, it should be 2022-2024

File: plugin/evm/codec.go [hunk 2]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2023-2024, Chain4Travel AG. All rights reserved.
```
Comment (mo-c4t) [inline plugin/evm/codec.go:1]: year is not correct for c4t license header, it should be 2022-2024

File: plugin/evm/service.go [hunk 3]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2023-2024, Chain4Travel AG. All rights reserved.
```
Comment (mo-c4t) [inline plugin/evm/service.go:1]: year is not correct for c4t license header, it should be 2022-2024

File: accounts/abi/bind/backends/simulated.go [hunk 4]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2023-2024, Chain4Travel AG. All rights reserved.
```
Comment (mo-c4t) [inline accounts/abi/bind/backends/simulated.go:1]: year is not correct for c4t license header, it should be 2022-2024

File: internal/ethapi/backend.go (hunk 1 above)
Comment (evlekht) [reply internal/ethapi/backend.go:1]: that was this way in other dev branches as well, I think. Lets finish Berlin merges, than add license linter to workflows and update this systematically.

**Review by mo-c4t — DISMISSED** [review]

**Review by mo-c4t — APPROVED** [review]

**Pull request merged**